pkg/metrics/collector/daemon.go

```diff
@@ -7,13 +7,22 @@
 package collector
 
 import (
+	"github.com/containerd/nydus-snapshotter/pkg/daemon/types"
 	"github.com/containerd/nydus-snapshotter/pkg/metrics/data"
 )
 
 type DaemonEventCollector struct {
 	event string
 }
 
+type DaemonInfoCollector struct {
+	Metrics *types.DaemonInfo
+}
+
 func (d *DaemonEventCollector) Collect() {
 	data.NydusdEventCount.WithLabelValues(d.event).Inc()
 }
+
+func (d *DaemonInfoCollector) Collect() {
+	data.NydusdVersion.WithLabelValues(d.Metrics.Version.PackageVer).Inc()
```

Comment:
Perhaps, we should use Gauge instead. When snapshots are removed, nydusd stops, and the number of running nydusd decreases. In addition, we should also consider the gauge should change when a nydusd dies in an accident and when nydus-snapshotter restarts, it re-manage the running nydusd daemons.